ccm_web/client/src/pages/GradebookCanvas.tsx

```diff
@@ -181,7 +182,9 @@ function ConvertCanvasGradebook (): JSX.Element {
   }
 
   const handleNoLetterGradesError = (): void => {
-    setPageState({ state: GradebookCanvasPageState.InvalidUpload, errorMessage: [<Typography key='0'><Link href='https://community.canvaslms.com/t5/Instructor-Guide/How-do-I-enable-a-grading-scheme-for-a-course/ta-p/1042' target='_new' rel='noopener'>Grading Scheme in settings</Link> needs to be check marked for letter grade to appear in the CSV file.</Typography>] })
+    const { canvasURL, course } = props.globals
+    const settingsURL = `${canvasURL}/courses/${course.id}/settings`
```

Comment:
Do you think we should link to the page in general, or more specifically

const settingsURL = `${canvasURL}/courses/${course.id}/settings#course_grading_standard_enabled`

Reply:
I thought about this, what if canvas changes the name at some point. I really don't know how often that happen so I left it generic. what you think? I can do either way but don't know what is best approach here

Reply:
@jonespm what is your opinion on giving a direct link https://umich.beta.instructure.com/courses/403334/settings#course_grading_standard_enabled to a form option on canvas? 
I feel that these things could be easily renamed then these links will break and we need to have a hot fix for it.

Reply:
I wondered that too.. it seems that when you have a non-existent anchor it just goes to the main page as if you didn't append one.  That seems fine, though it could break and nobody would notice.

It seems safe to add it on there since it can't really break, but it does introduce a dependency outside of our control that is not easy to keep track of.

I don't know if it's particularly important one way or the other..

Reply:
@chrisrrowland I changed it as per your recommendation but it is not going to that exact location as we wanted. Infant it is default to setting APP tab.

Reply:
Hmm.. so for me its resolving to https://canvas-test.it.umich.edu/courses/483458/settings#course_grading_standard_enabled and seems to work as expected

Reply:
It's weird that it is not happening in Beta env like you described https://umich.beta.instructure.com/courses/403334/settings#course_grading_standard_enabled

Reply:
Done

Reply:
For posterity, the behavior here seems browser dependent and mildly inconsistent/infuriating.

Sometimes it works, sometimes it doesn't, but it seems like the best case is a slightly more user friendly experience, and the worst case is that it doesn't do anything.